utils/frame/rpc/support/Cargo.toml

```diff
@@ -23,4 +23,5 @@ sc-rpc-api = { version = "0.10.0-dev", path = "../../../../client/rpc-api" }
 
 [dev-dependencies]
 frame-system = { version = "4.0.0-dev", path = "../../../../frame/system" }
+scale-info = "0.10.0"
```

Comment:
Is this dependency actually used?

Reply:
Yes it is required for a doc test which uses `decl_module`, which requires the `scale-info` dependency.